ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -133,6 +133,13 @@ class ReactIterableAPI: RCTEventEmitter {
         resolver(IterableAPI.userId)
     }
     
+    @objc(registerDeviceToken:)
+    func registerDeviceToken(token: String?) {
+        ITBInfo()
+
+        IterableAPI.registerDeviceToken(token)
```

Comment:
As discussed, we might have to either convert string to Data. or Have function which accepts String

Reply:
I updated this call to convert to Data before passing it to the native layer.